doc/user-guide/usage.md

```diff
@@ -0,0 +1,380 @@
+## Deploying LVMCluster CR
+
+This guide assumes you followed steps in [Readme][repo-readme] and LVM operator
+(hereafter LVMO) is running in your cluster.
+
+Below are the available disks in our test kubernetes cluster node and there are
+no existing LVM Physical Volumes, Volume Groups and Logical Volumes
+
+``` console
+sh-4.4# lsblk
+NAME    MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
+sdb       8:16   0 893.8G  0 disk
+|-sdb1    8:17   0     1M  0 part
+|-sdb2    8:18   0   127M  0 part
+|-sdb3    8:19   0   384M  0 part /boot
+`-sdb4    8:20   0 893.3G  0 part /sysroot
+sr0      11:0    1   987M  0 rom
+nvme0n1 259:0    0   1.5T  0 disk
+nvme1n1 259:1    0   1.5T  0 disk
+nvme2n1 259:2    0   1.5T  0 disk
+sh-4.4# pvs
+sh-4.4# vgs
+sh-4.4# lvs
+```
+
+Here LVMO is installed in `lvm-operator-system` namespace via `make deploy`
```

Comment:
Instead of `make` commands, upstream users generally appreciate either `kubectl` or `helm` commands for install. The `make` is just a black box to them. `make` works well during development, just not for production.

Reply:
- ack, this'll be a change in repo readme, you might want to have a look at #99  as well 🤔 
- the change more or less translates to push an image to the registry on every release and update yaml manifest for the same like `IMG=<pushed-image> make deploy > examples/lvmo.yaml`, at best `kubectl -f`, `helm` is a stretch atm
- will take a note of it and update the instructions when we push image to upstream accessible registry

Reply:
- Will be taken up in a separate PR